ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2

```diff
@@ -16,4 +16,25 @@ management api gnmi
 {%     if management_api_gnmi.octa is arista.avd.defined %}
```

Comment:
Need to fix this. This should be using defined test instead of arista.avd.defined. At least this needs to be fixed in 4.0

Reply:
Or change the logic to use arista.avd.defined(true)

Reply:
this is not a boolean in the old data model. It was defined in a vague (wrong) way, where simply setting the key to null, would enable the feature. We need to keep this behavior for now. I have updated to `defined` filter.

Reply:
Thank you. That should generate a new artifact, prints the eos-native command.